apps/cnquery/cmd/status.go

```diff
@@ -135,6 +70,81 @@ Status sends a ping to Mondoo Platform to verify the credentials.
 	},
 }
 
+func checkStatus() (Status, error) {
+	s := Status{
+		Client: ClientStatus{
+			Timestamp: time.Now().Format(time.RFC3339),
+			Version:   cnquery.GetVersion(),
+			Build:     cnquery.GetBuild(),
+		},
+	}
+
+	opts, optsErr := config.Read()
+	if optsErr != nil {
+		return s, cli_errors.NewCommandError(errors.Wrap(optsErr, "could not load configuration"), 1)
+	}
+
+	httpClient, err := opts.GetHttpClient()
+	if err != nil {
+		return s, cli_errors.NewCommandError(errors.Wrap(err, "failed to set up Mondoo API client"), 1)
+	}
+
+	sysInfo, err := sysinfo.Get()
+	if err == nil {
+		s.Client.Platform = sysInfo.Platform
+		s.Client.Hostname = sysInfo.Hostname
+		s.Client.IP = sysInfo.IP
+	}
+
+	// check server health and clock skew
+	upstreamStatus, err := health.CheckApiHealth(httpClient, opts.UpstreamApiEndpoint())
+	if err != nil {
+		log.Error().Err(err).Msg("could not check upstream health")
+	}
+	s.Upstream = upstreamStatus
+
+	latestVersion, err := cnquery.GetLatestVersion(httpClient)
+	if err != nil {
+		return s, cli_errors.NewCommandError(errors.Wrap(err, "failed to get latest version"), 1)
+	}
+
+	s.Client.LatestVersion = latestVersion
+
+	// check valid agent authentication
+	plugins := []ranger.ClientPlugin{}
+
+	// try to load config into credentials struct
+	credentials := opts.GetServiceCredential()
+	if credentials != nil && len(credentials.Mrn) > 0 {
+		s.Client.ParentMrn = credentials.ScopeMrn
```

Comment:
I am not sure if this breaks anything. What if I have an old service account that still has `ParentMrn` instead of `ScopeMrn`. My integration will be broken in that case and I would need to re-create my service account. I suggest not changing this now because we don't know what the impact will be for customers